core-bundle/assets/scripts/choices.js

```diff
@@ -0,0 +1,58 @@
+(function () {
+    'use strict';
+
+    const initializedChoices = new WeakMap();
+
+    const init = (select) => {
+        // Check if this select has already been initialized
+        if (initializedChoices.has(select)) {
+            return;
+        }
+
+        initializedChoices.set(select, true);
+
+        new Choices(select, {
+            shouldSort: false,
+            duplicateItemsAllowed: false,
+            allowHTML: false,
+            removeItemButton: true,
+            searchEnabled: select.options.length > 7,
+            classNames: {
+                containerOuter: ['choices', ...Array.from(select.classList)],
+                flippedState: ''
+            },
+            fuseOptions: {
+                threshold: 0.4,
```

Comment:
Do I have to add `includeScore: true,` here as well or will the properties be merged?

Reply:
Doesn't look like it as it only assigns other parameters but not some defaults if we overwrite fuseOptions.
https://github.com/Choices-js/Choices/blob/2424531a53464aa937d8b302b9300e3ffb1da223/public/assets/scripts/choices.js#L2965

We'd have to add `includeScore: true` as well